tests/test_switch.py

```diff
@@ -0,0 +1,16 @@
+from textual.app import App, ComposeResult
+from textual.widgets import Switch
+
+
+async def test_switch_click_doesnt_bubble_up():
```

Comment:
Can you please add a comment with a link to the issue?
We usually say something like "Regression test for <URL>".

Reply:
Thanks. You mentioned this in the meta issue, but for some reason I thought it referred to the changelog.

I added a docstring that references the original ticket.